src/plugins/visualizations/common/expression_functions/xy_dimension.ts

```diff
@@ -7,13 +7,35 @@
  */
 
 import { i18n } from '@kbn/i18n';
-import type { ExpressionValueVisDimension } from '../../../visualizations/public';
+import type { ExpressionValueVisDimension } from './vis_dimension';
+
 import type {
   ExpressionFunctionDefinition,
-  Datatable,
   ExpressionValueBoxed,
-} from '../../../expressions/public';
-import type { Dimension } from '../types';
+  Datatable,
+  DatatableColumn,
+  SerializedFieldFormat,
+} from '../../../expressions/common';
+
+export interface DateHistogramParams {
+  date: boolean;
+  interval: number;
+  intervalESValue: number;
+  intervalESUnit: string;
+  format: string;
+  bounds?: {
+    min: string | number;
+    max: string | number;
+  };
+}
+
+export interface HistogramParams {
+  interval: number;
+}
+
+export interface FakeParams {
+  defaultValue: string;
+}
```

Comment:
Is this a duplicate of the `vis_type_xy/public/types` content?

In case can we rely on these only without duplication? If you reexport these from the `param.ts` file the impact on other files should be minimal.

Reply:
I forgot them, thanx. I moved them to the visualizations plugin, otherwise we will have circular dependencies